src/LSP.hs

```diff
@@ -214,8 +214,8 @@ handlers f =
         sendNotification diagramAcceptMethod setSynthesis
         sendNotification diagramAcceptMethod (updateOptions file)
         (core, dflags) <- withRunInIO (\_u -> compileToCore file)
-        let ir = translateCoreProgram dflags core
-        let graphMessage = requestBounds file ir
+        let (forsydeIR, _lookupSignals) = translateCoreProgram dflags core
```

Comment:
Looking at the type change for `translateCoreProgram`, this should be enough for it to work. But the line number the CI gave us was nonsensical so I'm not sure if it merges into a temporary dev or something. After #258 this is also done closer to the main for the JSON output which might be why it fails.